spec/models/like_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject do
+    user = User.create(name: 'Haurchefant', photo: '', bio: 'In loving memory of our dear partner')
+    post = Post.new(author: user, title: 'Do not look at me so', text: 'A smile better suits a hero')
+    Like.new(author: user, post:)
+  end
+
+  before { subject.save }
+
```

Comment:
- [ ] Your specs are showing errors, kindly fix them for the next step.


SyntaxError:
  /Users/rustamjon/Rustamxon7/blog-appssss/spec/models/like_spec.rb:7: syntax error, unexpected ')'
  ...  Like.new(author: user, post:)
  ...                              ^

Reply:
Hello @Rustamxon7 , thanks for your review.

I ran the tests and the linters and I did not get any of those errors, would you mind explaining how you got them? 
![pic 4](https://user-images.githubusercontent.com/92347711/173163525-627606ef-489b-4691-b9a7-5f448cab2297.png)